frontend/encore/reactjs.rst

```diff
@@ -11,7 +11,7 @@ Using React? First add some dependencies with Yarn:
 .. code-block:: terminal
 
     $ yarn add @babel/preset-react --dev
-    $ yarn add react react-dom prop-types
+    $ yarn add react react-dom prop-types --dev
```

Comment:
That's funny because we have the same contribution last week (https://github.com/symfony/symfony-docs/pull/14867) 

Are we sure about this modification?

Reply:
I thought we did EVERYTHING as `--dev` because *none* of this is actually read at runtime. This allows for the edge case of an application where you have Encore + some Node app that you *do* run on production at runtime..

But, I'm going to remove it so it doesn't get blocked.

Reply:
As you wish!